sbin/winbindd-wrapper

```diff
@@ -127,6 +129,7 @@ start();
 cleanup();
 
 END {
+    &Systemd::Daemon::notify( STOPPING => 1 );
```

Comment:
Notify should be done only when you are doing the delete pid.
To avoid calling notify if you run the help